build.yaml

```diff
@@ -529,6 +529,18 @@ steps:
     - deploy_batch
     - deploy_batch_pods
     - test_batch_image
+ - kind: runImage
+   name: setup_pipeline
+   image:
+     valueFrom: test_pipeline_image.image
+   alwaysRun: true     
+   script: |
+     set -e
+     bucket=(python3 -c 'import hailjwt as hj; print(hj.unsafe_decode_from_file($HAIL_TOKEN_FILE)["bucket"])')
+     gcloud -q auth activate-service-account --key-file=/gsa-key/privateKeyData
+     gsutil -m cp -r /test/resources/  gs://${bucket}/{{ token }}/pipeline/
```

Comment:
How do you get HAIL_TOKEN_FILE here?  I say just hardcode the hail|batch-test user bucket here.  You will need to mount the gsa-key, hardcode that, too.